vpr/src/base/stats.cpp

```diff
@@ -364,7 +364,7 @@ void print_wirelen_prob_dist() {
                 VTR_LOG("Realloc'ing to increase 2-pin wirelen prob distribution array.\n");
                 incr = index - prob_dist_size + 2;
                 prob_dist_size += incr;
-                prob_dist = (float*)vtr::realloc(prob_dist, prob_dist_size * sizeof(float));
+                prob_dist = new float[prob_dist_size];
```

Comment:
This looks like a memory leak.  realloc is a combination of free + malloc; I think you need a new, copy, delete here.